sdk/keyvault/azure-security-keyvault-secrets/inc/azure/keyvault/secrets/secret_client.hpp

```diff
@@ -256,13 +257,27 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Secrets {
      * Restores a backed up secret, and all its versions, to a vault.
      * This operation requires the secrets/restore permission.
      *
-     * @param backup The backup payload as base64url encoded vector of bytes.
+     * @param backup The backup payload as encoded vector of bytes.
      * @param context The context for the operation can be used for request cancellation.
      *
      * @return The Secret wrapped in the Response.
      */
     Azure::Response<KeyVaultSecret> RestoreSecretBackup(
-        BackupSecretResponse const& backup,
+        BackupSecretResult const& backup,
```

Comment:
As mentioned above, there's no practical reason to have this. Let's not add overloads without purpose since we'll have to support them forever.

Reply:
done